src/app/store/services/index.js

```diff
@@ -0,0 +1,14 @@
+import { rest } from '@/app/global/services'
+
+export async function getResource (url) {
+  return rest.get(url).then(res => res.data)
```

Comment:
if only, lol, right???

Reply:
It would make things a lot easier. Might just have to learn how to write up REST endpoints in PHP/Laravel...